content/pl/docs/setup/_index.md

```diff
@@ -0,0 +1,112 @@
+---
+no_issue: true
+title: Od czego zacząć
+main_menu: true
+weight: 20
+content_template: templates/concept
+card:
+  name: setup
+  weight: 20
+  anchors:
+  - anchor: "#learning-environment"
+    title: Środowisko do nauki
+  - anchor: "#production-environment"
+    title: Środowisko produkcyjne  
+---
+
+{{% capture overview %}}
+
+Ten rozdział poświęcony jest różnym metodom konfiguracji i uruchomienia Kubernetesa.
+
+Istnieje wiele rozwiązań dopasowanych do różnych potrzeb użytkowników: łatwości w utrzymaniu, wymagań bezpieczeństwa, poziomu sterowania, dostępności zasobów oraz niezbędnego doświadczenia do zarządzania klastrem.
+
+Klaster Kubernetes możesz zainstalować na lokalnym komputerze, w chmurze czy w prywatnym centrum obliczeniowym albo skorzystać z klastra Kubernetes udostępnianego jako usługa. Inną możliwością jest budowa własnego rozwiązania opartego o różnych dostawców usług chmurowych, bądź bazującego bezpośrednio na sprzęcie fizycznym.
```

Comment:
Trochę masło maślane ;)

Reply:
Taaa... ale nie mam pomysłu. To trzeba by przeredagować zupełnie jak już.

Reply:
> You can deploy a Kubernetes cluster on a local machine, cloud, on-prem datacenter; or choose a managed Kubernetes cluster. You can also create custom solutions across a wide range of cloud providers, or bare metal environments.

Reply:
@nvtkaszpir Any ideas to change? Or leave it as it is?

Reply:
Może tak? 
```suggestion
Klaster Kubernetes możesz uruchomić na lokalnym komputerze, w chmurze lub w prywatnym centrum obliczeniowym. Możesz również skorzystać z klastra Kubernetes udostępnianego jako usługa. Jeszcze inną możliwością jest budowa własnego rozwiązania opartego o różnych dostawców usług chmurowych, bądź bazującego bezpośrednio na sprzęcie fizycznym.
```

Reply:
I'm not happy with "uruchomić" - it's slightly different meaning here.